README.md

```diff
@@ -19,15 +19,24 @@ not be used in production.
 
 Ensure that Docker is installed and running on your machine before proceeding.
 
-#### Using Docker Compose File
-1. Create a `logging.ini` file from the example in the root of the project directory:
+1. Create a `.env` file alongside the `.env.example` file. Use the example file as a reference and modify the values accordingly.
+    ```bash
+    cp .env.example .env
+    ```
+
+2. Create a `logging.ini` file alongside the `logging.example.ini` file. Use the example file as a reference and modify it accordingly:
     ```bash
-   cp logging.example.ini logging.ini
+    cp logging.example.ini logging.ini
     ```
 
-2. Build and start the Docker containers:
+3. (**Required only if JWT Auth is enabled**) Inside the `keys` directory at the root of the project directory, create a copy of the public key generated by the authentication component. This is needed for decoding of JWT access tokens signed by the corresponding private key.
```

Comment:
This should mention creating the keys directory like ldap-jwt-auth as it is no longer committed.

Reply:
Sorry I forgot to revert the changes for this. Addressed now in https://github.com/ral-facilities/inventory-management-system-api/pull/169/commits/98866f6433220d558752f7356775ae96515d920c.